client/src/pages/AccountInfo.jsx

```diff
@@ -53,7 +56,7 @@ const handleFormSubmit = async (event ) => {
             variables: { ...userFormData, id: Auth.getProfile().data._id },
         });
         const { token } = response.data.updateUser;
```

Comment:
This line can be deleted as well.

Reply:
56? or 58?

Reply:
const { token } = response.data.updateUser;